src/mangosd/CliRunnable.cpp

```diff
@@ -135,7 +135,7 @@ bool ChatHandler::GetDeletedCharacterInfoList(DeletedInfoList& foundList, std::s
     {
         if (resultChar->GetRowCount() > 100)
         {
-            PSendSysMessage("Too many results %llu. Narrow it down.", resultChar->GetRowCount());
+            PSendSysMessage("Too many results %lu. Narrow it down.", resultChar->GetRowCount());
```

Comment:
This one is also a bit suspect, as it is dependent on the environment (for example on Linux, `lu` is equivalent to uint64, on MacOS `llu` is equivalent to uint64, so I'm guessing this one should probably remain llu and we'll just ignore that particular warning)

Reply:
I suggest to cast the result to uint32 and use %u format